src/top_model/main.cpp

```diff
@@ -209,5 +219,17 @@ std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP = std::make_share
     r.run_until(NDTime("04:00:00:000"));
     auto elapsed = std::chrono::duration_cast<std::chrono::duration<double, std::ratio<1>>>(hclock::now() - start).count();
     cout << "Simulation took:" << elapsed << "sec" << endl;
+
+    /**
+    * Convert the simulator output into required format
+    */
+    convert_output(sim_input, sim_output);
+
+    /**
+    * Part E
+    * Convert the simulator output to phase wise time elapsed
+    */
+    phase_observer(sim_output, phase_output);
```

Comment:
when I execute ABP.exe, a set of values are getting printed on the console. I don't understand why.

Reply:
![1111](https://user-images.githubusercontent.com/53232445/63641327-5a9c1900-c67a-11e9-8390-54ad65138fa2.png)

Reply:
i have absolutely no idea why they are printed. i don't get any printed values.

Reply:
> when I execute ABP.exe, a set of values are getting printed on the console. I don't understand why.

Hi, I have removed the unwanted cout commands which were printing values on console. Thanks.
![image](https://user-images.githubusercontent.com/53234950/63644306-feee8180-c6b3-11e9-88fd-af362a7e6fc7.png)